code/src/index.css

```diff
@@ -1,13 +1,234 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #191414;
+  margin: 5px;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+.container {
+  width: 300px;
+  margin: 0 auto;
+  padding: 40px 0;
+}
+
+.menu {
+  position: fixed;
+  top: 0;
+  right: 10px;
+  width: 25px;
+  filter: invert();
+  z-index: 2;
+  margin: 20px 10px;
+}
+
+.menu:hover {
+  width: 30px;
+  transition: 0.3s;
+}
+
+.title {
+  margin: 10px 5px;
+  color: #ffffff;
+  font-size: 25px;
+  border-bottom: 1px solid rgb(55, 55, 55);
+  padding-bottom: 5px;
+}
+
+.album-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.album-grid + .title:last-of-type {
+  padding-top: 50px;
+}
+
+.album-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  row-gap: 40px;
+}
+
+.album-container {
+  justify-self: center;
+}
+
+/* ALBUM COVER */
+.image-container {
+  position: relative;
+}
+
+.album-image {
+  width: 100%;
+}
+
+/* ALBUM NAME */
+.album-name {
+  color: #ffffff;
+  font-size: 14px;
+  font-weight: bold;
+  margin: 4px 0;
+}
+
+/* ARTIST NAME */
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+  display: inline;
+}
+
+a {
+  text-decoration: inherit;
+  color: inherit;
+}
+
+.artist-name::after {
+  content: ', ';
+}
+
+.artist-name:last-child::after {
+  content: '';
+}
+
+.artist-name:nth-last-of-type(2)::after {
+  content: ' & ';
+}
+
+.artist-name:hover,
+.album-name:hover,
+.playlist:hover {
+  text-decoration: underline;
+}
+
+/* ICONS */
+
+.icons-container {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 40% 0;
+}
+
+.icon {
+  display: none;
+  padding: 0 15px;
+  filter: invert();
+  z-index: 1;
+}
+
+.play {
+  width: 50px;
+}
+.dots {
+  width: 20px;
+}
+.heart {
+  width: 25px;
+}
+
+.play:hover {
+  transform: scale(1.3);
+  transition: 0.3s;
+  padding: 0 10px;
+}
+
+.icons-container:hover {
+  background: rgba(41, 42, 47, 0.5);
+  transition: 0.3s;
+}
+
+.icons-container:hover .icon {
```

Comment:
We noticed that the icons move closer together when you hover on the play button. How did you achieve this effect?